src/Controls/samples/Controls.Sample.UITests/Issues/Issue18716.xaml

```diff
@@ -0,0 +1,12 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage 
+    xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+    xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml" 
+    x:Class="Maui.Controls.Sample.Issues.Issue18716">
+  <Grid>
+    <WebView
+      x:Name="WaitForStubControl"
+      Source="https://freetestdata.com/wp-content/uploads/2021/09/Free_Test_Data_1MB_PDF.pdf">
```

Comment:
I think it's better to maybe add a local pdf that would make sure we don't hit any connection issues loading this

Reply:
Yeah, we can maybe at least host the pdf in the maui repo since we cannot control what this pdf is/does in the future.